File: 62750938-main/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

void left_side(int spaces, int bricks);
void right_side(int bricks);

int main(void)
{

    int number = 0;
    do
    {
        number = get_int("What's the size? ");
    }
    while (number < 1 || number > 8);

    for (int j = 1; j <= number; j++)
    {
        left_side(number - j, j);
        right_side(j);
    }
}

void left_side(int spaces, int bricks)
{
    for (int i = 0; i < spaces; i++)
    {
        printf(" ");
    }
    for (int k = 0; k < bricks; k++)
    {
        printf("#");
    }
    printf("  ");
}

void right_side(int bricks)
{
    for (int b = 0; b < bricks; b++)
    {
        printf("#");
    }
    printf("\n");
}

File: 62750938-main/sentimental-mario-more/mario.py
def left_side(spaces, bricks):
    for i in range(spaces):
        print(" ", end="")
    for i in range(bricks):
        print("#", end="")

    print("  ", end="")


def right_side(bricks):
    for i in range(bricks):
        print("#", end="")

    print("")


while True:
    try:
        number = int(input("Size: "))
        if number > 0 and number < 9:
            break

    except ValueError:
        pass


for i in range(number):
    left_side((number - i) - 1, i + 1)
    right_side(i + 1)
